tsconfig.json

```diff
@@ -33,5 +33,9 @@
     "config.js",
     "tmp"
   ],
-  "ts-node": { "transpileOnly": true, "esm": true }
+  "ts-node": {
+    "transpileOnly": true,
+    "esm": false,
```

Comment:
`esm=true` breaks `yarn start` command, and everything seems to be working well without it